src/query/pipeline/sources/src/processors/sources/input_formats/input_format.rs

```diff
@@ -30,6 +30,8 @@ pub trait InputFormat: Send + Sync {
 
     fn default_field_delimiter(&self) -> u8;
 
+    fn default_nan_display(&self) -> String;
```

Comment:
we do not need this in the trait InputFormat, nan is used only in FieldDecoder.
for csv/tsv, InputFormat is used for cutting rows and fields with delimiter/quote/escape. these bytes of fields is feed to FieldDecoder for interpretation.

Reply:
default_field_delimiter is also temporary,  need  refactor later:
 maybe add a enum to hold used options in XXXInputFormat.

Reply:
Maybe we can add a display dictionary to do it instead of dealing with 'default_nan_display' for `nan`.

Reply:
To avoid the logic and codes fat and fat, we should do the common(For example, are there other displays that also have such a need?) instead add more special cases.

Reply:
yes, as I commented above

1.  default_nan_display is useless here
2. default_field_delimiter will be removed from this trait later.